.github/workflows/test.yml

```
@@ -6,6 +6,22 @@ env:
   npm_config_cache: /tmp/npm-cache
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v4
+      - name: "Use Node.js"
+        uses: actions/setup-node@v4
+        with:
+          node-version: 23
```

Please use an LTS version.

```suggestion
          node-version: 'lts/*'
```

---

Nice. I didn't know this option :rocket: